main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+)
+
+func main() {
+	// TODO: task #1 - Why is it not working?
+	// task #1 failed because the return value from os.Getenv is not constant.
+	// by assigning var insted of const will fix this issue
+	var task1 = os.Getenv("task_1")
+	if !reflect.DeepEqual("", task1) {
+		fmt.Println("Task #1 failed!")
+	}
```

Comment:
awesome! Do you know why it couldn't accept `const` ?

Reply:
probably because the const is used for the  input that cannot be modified later and the return value of function os.Getenv required as string that not match with const